config/config.properties

```diff
@@ -97,6 +97,14 @@ pravegaservice.dataLog.impl.name=BOOKKEEPER
 # Default value: HDFS
 # pravegaservice.storage.impl.name=HDFS
 
+
+# Storage layout for Tier 2 storage.
+# Valid values:
+#   CHUNKED_STORAGE - Using ChunkedSegmentStorage.
```

Comment:
Clearly indicate this is EXPERIMENTAL and not to be used in production.

Reply:
fixed.